submissions/AnnaStoyano/pre-oop/index.js

```diff
@@ -0,0 +1,89 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Lucy',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'guf',
+   friends: ['Luck']
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Murzick',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'meow',
+   friends: ['Elza','Nicky']
+};
+
+const woman = {
+   species: 'human',
+   name: 'Anna',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hello!',
+   friends: ['Sergey','Kate','Karina']
+}
+
+const man = {
+   species: 'human',
+   name: 'Vlad',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi!',
+   friends: ['Sergey','Anton','Vlad']
+}
+
+const catWoman = {
+   species: 'human',
+   name: 'Liza',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+   friends: ['Batman']
+};
+
+
+const inhabitants = [man,woman,cat,dog,catWoman];
+let output = '';
+inhabitants.forEach(item=>{
+   for(let key in item){
+      output+=`<b>${key}</b>: ${item[key]}\t`
+   }
+   output+='\n'
```

Comment:
This fragment relies on:
1. order of object properties (which is not guaranteed)
2. there are no extra properties we do not want in output

Both issues can be resolved by specifying the list of keys and mapping it into `<b>${key}</b>: ${item[key]}\t`.

Besides, after having keys mapped into strings they can be joined with `\t`, and strings per object then subsequently can be joined with `\n`.

In other words you might use `.map`, `.join` also nested (map+join per each object, map+join per key list for each object).